releasenotes/README.md

```diff
@@ -16,6 +16,10 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
+* NEW: Some commands (`force:org:open`, `force:org:display`, `force:user:display`, `force:source:open`) output tokens and URLs with tokens embedded in them.  They now include a warning about the risks of sharing that sensitive information. 
+
+* NEW: `force:source:retrieve -m CustomField` (or the manifest equivalent using `*` for CustomField without CustomObject entries) succeeds but returns no results.  It's a confusing Metadata API quirk.  Now, when you run the command like that, it will output a warning with instructions for how to do what you probably meant to do.
```

Comment:
```suggestion
* NEW: `force:source:retrieve -m CustomField` (or the manifest equivalent using `*` for CustomField without CustomObject entries) succeeds but returns no results.  It's a confusing Metadata API quirk.  Now, the command outputs a warning with instructions for how to modify so that you see the results.
```

Reply:
It's not that you can't see the result, but that there is nothing retrieved/returned.  Is that how it reads?

Reply:
Ah. ok. I was unsure what "probably mean to do" meant. What does the warning display? What are the instructions?